macros/get_project_columns.sql

```diff
@@ -6,8 +6,6 @@
     {"name": "archived", "datatype": "boolean"},
     {"name": "color", "datatype": dbt_utils.type_string()},
     {"name": "created_at", "datatype": dbt_utils.type_timestamp()},
-    {"name": "current_status", "datatype": dbt_utils.type_string()},
-    {"name": "due_date", "datatype": dbt_utils.type_timestamp()},
```

Comment:
Why are we removing these fields? They don't seem to be depreciated. Or are they?